gradle.properties

```diff
@@ -1,7 +1,7 @@
 android.useAndroidX=true
 
 GROUP=com.tinder.scarlet
-VERSION_NAME=0.1.13-SNAPSHOT
+VERSION_NAME=0.1.14-SNAPSHOT
```

Comment:
just a heads up, this won't be used to generate the artifact version. The version will be determined by the release name

Reply:
Ah okay, good point
So this is irrelevant